src/components/navigation/sw-tabs/sw-tabs.interactive.stories.js

```diff
@@ -76,195 +77,189 @@ const tabItems = [
 ]
 
 export default {
-  ...defaultTabsStory,
+  ...meta,
   title: 'Interaction Tests/Navigation/sw-tabs',
-  component: SwTabs,
 };
 
-const Template = (args, { argTypes }) => ({
-  props: Object.keys(argTypes),
-  components: { SwTabs },
-  template: `
-  <sw-tabs v-bind="$props"></sw-tabs>`
-});
+export const  VisualTestRenderTabs = {
+  name: 'Render tabs',
+  args: {
+    small: false,
+    defaultItem: 'item2',
+    items: tabItems.slice(0,2)
+  }
+}
 
-export const VisualTestRenderTabs = Template.bind({});
-VisualTestRenderTabs.storyName = 'Render tabs';
-VisualTestRenderTabs.args = {
-  ...Default.args,
-  small: false,
-  defaultItem: 'item2',
-  items: tabItems.slice(0,2)
-};
+export const VisualTestRenderTabsVertical = {
```

Comment:
I have a guess why the snapshot could differ from the previous one. Before we used the `Default` args as the base which uses `small: true`. Now it uses `meta` which has `small: false` as default. Then the change in the snapshot would make sense

Reply:
Yes this was the issue.